aspnet/publishing/linuxproduction.rst

```diff
@@ -0,0 +1,190 @@
+
+    If you plan to install optional Nginx modules you may be required to
+    build Nginx from source.
+
+We are going to ``apt-get`` to install Nginx. The installer also creates a System V init script that runs Nginx as daemon on system startup. Since we just installed Nginx for the first time, we can explicitly start it by running
+
+.. code-block:: bash
+
+    sudo service nginx start
+
+At this point you should be able to navigate to your browser and see the default landing page for Nginx.
+
+Configure Nginx
+~~~~~~~~~~~~~~~
+
+We will now configure Nginx as a reverse proxy to forward requests to our ASP.NET application
+
+We will be modifying the ``/etc/nginx/sites-available/default``, so open it up in your favorite text editor and replace the contents with the following.
+
+.. code-block:: nginx
+   :emphasize-lines: 7
+
+    server {
+        listen 80;
+        location / {
+            proxy_pass http://unix:/var/aspnet/HelloMVC/kestrel.sock;
+            proxy_http_version 1.1;
+            proxy_set_header Upgrade $http_upgrade;
+            proxy_set_header Connection keep-alive;
+            proxy_set_header Host $host;
+            proxy_cache_bypass $http_upgrade;
+        }
+    }
+
+This is one of the simplest configuration files for Nginx that forwards incoming public traffic on your port ``80`` to a unix socket that your web application will listen on. You can specify this Unix socket in your ``project.json`` file.
+
+.. code-block:: json
+    :caption: project.json (truncated)
+
+    "commands": {
+        "web": "Microsoft.AspNet.Server.Kestrel --server.urls http://unix:/var/aspnet/HelloMVC/kestrel.sock",
+    },
+
+.. note::
+
+    The ``proxy_set_header Connection keep-alive;`` is **required** as a temporary workaround a `known bug in Kestrel <https://github.com/aspnet/KestrelHttpServer/issues/341>`_.
+
+You might want to look at ``etc/nginx/nginx.conf`` to make other necessary changes to your nginx environment to configure your nginx environment.
+
+Once you have completed making changes to your nginx configuration you can run ``sudo nginx -t`` to verify the syntax of your configuration files. If you see any no errors you can ask nginx to pick up the changes by running ``sudo nginx -s reload``.
+
+
+
+Monitoring our Web Application
+------------------------------
+
+Nginx will forward requests to your Kestrel server, however unlike IIS on Windows, it does not mangage your Kestrel process. In this tutorial, we will use `supervisor <http://supervisord.org/>`_ to start our application on system boot and restart our process in the event of a failure.
+
+
+Installing supervisor
+~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: bash
+
+    sudo apt-get install supervisor
+
+.. note::
+
+    ``supervisor`` is a python based tool and you can acquire it through `pip <http://supervisord.org/installing.html#installing-via-pip>`_ or `easy_install <http://supervisord.org/installing.html#internet-installing-with-setuptools>`_ instead.
```

Comment:
I already have it in there https://github.com/aspnet/Docs/pull/1014#discussion-diff-53977152R120